### ISSUE TITLE
Option to disable permissions per form

### ISSUE THREAD

**Issue by benoitblancquaert** [verbb/formie#1426]
### What are you trying to do?

It would be great if you could configure to disable the permissions per form. The user groups quickly become polluted by Formie "content" and have to be updated after adding forms, which is quite annoying.

### What's your proposed solution?

Add an additional configuration option.

**Comment by engram-design** [comment]
Can you clarify what you mean by "disable permissions"? Is managing them via user groups not working for you? Keen to hear your use-case so I can get a good idea of what you're after. In particular, how and where these permissions should be managed.

I will mention that if you have "Manage all forms" permission enabled, then new forms are granted permission automatically.

It's a bit tricky to handle permissions on user-created content.

**Comment by bartrylant** [comment]
I think @benoitblancquaert is having the same issue as I have. I developed a site which has +100 forms. With the "Manage all forms" permission enabled, every user sees every form (and all submissions for every form). When you don't enable this option, you have to edit permissions for each new form you create.

I have a user group which should be able to manage forms and see submission in, for instance, a "transfers" category. I have another user group where people should see all forms in a "licenses" category, and a third user group where people should see all forms in a "contact forms" category. I should be able to put forms in a category (or this can also be the used template) so I can just assign people to user groups that can manage all past and future forms _in that certain category_. Just like you can assign user groups to different sections in Craft.

**Comment by bartrylant** [comment]
<img width="439" alt="Scherm­afbeelding 173" src="https://github.com/verbb/formie/assets/1434451/793a3290-45da-4e88-ae74-dfd4d6c1f88d">

**Comment by engram-design** [comment]
Right, so that would probably tie into the concept of [form groups](https://github.com/verbb/formie/issues/1039) as we'd need a concept of grouping forms in some way so you can define your permissions that way.

**Comment by benoitblancquaert** [comment]
Sorry @engram-design, my initial post was a bit short.

I'll explain my remarks based on a project I'm working on. I have a multi-site Craft setup with 20 different sites: 1 main site and 19 subsites. For each site there is a user group with the correct permissions. The 19 subsites use user groups with less rights. The Formie permissions are always:
- General / Access Formie
- Form / View forms
- Formie / View submissions
- Formie / View send notifications

**1st comment:**
The user groups are regularly updated. The site is still under development and sections are being added. In my opinion, maintaining the user groups would be easier if no user-created content was added to the permissions.

**2nd comment:**
The permissions per form (user-created content) is useful, but if forms are created by users, user groups must also be adjusted by the administrator.

**3rd comment:**
Ideally, Craft site A should not see forms in the Formie dashboard of Craft site B. I think this problem should indeed be addressed by adding (Formie) form groups and then linking them to the correct (Craft) user group.

**4th comment:**
Ideally, Craft site A should not see forms from Craft site B or others when selecting a form via a Formie field.

So my initial post is really only about comment 1 and 2, but I also think 3 and 4 are very important.

**Comment by engram-design** [comment]
I'll definitely admit that user group permissions are a bit strange when it comes to forms, as they are elements. It'd be similar to individual entries allowing permissions on them, which would be a nightmare, and pretty odd.

If you have "Manage all forms" permission enabled, then new forms are granted permission automatically. That means that even for a user with limited permissions to forms, so long as they have permissions to create a form, they'll always be able to access their own created forms.

The way around this to manage things better is some form or Form Group collection, which I've touched on in another comment. That way, there's a layer to categorise forms in your scenario a little more fine-grained.

And yes, there's a lack of multi-site support that would also make this easier, but adding multi-site support now before some Craft field infrastructure would be pushing the limits of the database engine too much.

The only other alternative is to roll our own, dedicated permissions system for forms.

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.7](https://github.com/verbb/formie/releases/tag/4.0.0-beta.7). User permissions have been given an overhaul with the addition of Form Groups.
